### PR TITLE
Add initial version of rad-plugins helm chart

### PR DESCRIPTION
**Pull request by mateuszdyminski**
Towards: https://linear.app/rad-security/issue/ENG-2109/remove-ksoc-name-and-replace-it-with-rad-in-all-plugins

### CODE REVIEW

File: scripts/modify_chart_for_aws_marketplace.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILE="./stable/rad-plugins/values.yaml"
+
+GCR_REGISTRY_NAME="us.gcr.io/ksoc-public"
```
Comment (maximillianbrain1) [inline scripts/modify_chart_for_aws_marketplace.sh:5]: We should replace this with our new ECR registry name

File: stable/rad-plugins/README.md.gotmpl [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+
+For example, the wildcard rule:
+```yaml
+runtime:
+  exporter:
+    execFilters:
+    - wildcard: "command * secret=(*)"
+```
+
+result in the following redacted commands:
```
[the remark this answers is not in the document]
Comment (mateuszdyminski) [reply stable/rad-plugins/README.md.gotmpl]: fixed

File: stable/rad-plugins/README.md.gotmpl [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+If you have enabled the `rad-runtime` plugin you should also see the following pods in a state of Running:
+
+```bash
+NAME                READY   STATUS    RESTARTS   AGE
+rad-runtime-7jjzg   2/2     Running   0          1m
+rad-runtime-p45g6   2/2     Running   0          1m
+rad-runtime-pmgtf   2/2     Running   0          1m
+```
+
+The number of pods below should equal the number of nodes in your cluster
```
[the remark this answers is not in the document]
Comment (mateuszdyminski) [reply stable/rad-plugins/README.md.gotmpl]: fixed

File: stable/rad-plugins/README.md.gotmpl [hunk 4]
```diff
@@ -0,0 +1,435 @@
+# RAD Security Plugins
+
+## Introduction
+
+This chart deploys the following plugins required for the [RAD Security](https://rad.security/) platform(former KSOC):
```
[the remark this answers is not in the document]
Comment (mateuszdyminski) [reply stable/rad-plugins/README.md.gotmpl]: fixed

File: scripts/modify_chart_for_aws_marketplace.sh (hunk 1 above)
Comment (mateuszdyminski) [reply scripts/modify_chart_for_aws_marketplace.sh]: fixed

**Review by pancernik — APPROVED** [review]

**Pull request merged**